tchannel/testing/vcr/cassette.py

```diff
@@ -0,0 +1,101 @@
+from __future__ import absolute_import
+
+from itertools import chain
+from collections import deque
+
+from . import types
+from . import yaml
+from .exceptions import RequestNotFoundError, UnsupportedVersionError
+
+
+# Version of the storage format.
+VERSION = 1
+
+
+class Cassette(object):
+    """Represents a series of recorded interactions."""
+
+    def __init__(self, path):
+        """Initialize a new cassette.
+
+        :param path:
+            File path at which this cassette will be stored.
+        """
+        self.path = path
+
+        # Number of times interactions from this cassette have been played.
+        self.play_count = 0
+
+        self._available = deque()
+        self._played = deque()
```

Comment:
It looks strange to me. data will take all three queues of 'interactions' and put them into file. But we it loads up them. These interactions are not classified as played, available, recorded any more. That's why I wonder if this _played is necessary. Because whey you load data from file, actually all are available ones, _played can actually be the _available.